File: Basic Problems/typescript/linear-search.ts
function linearSearch(arr: Array <number>, element: number ): object | string {
  for (let i:number = 0; i < arr.length; i++) {
    if (arr[i] === element) return {position: i, element}
  }
  return "Element not found in array"
}

const getElement = linearSearch([10, 20, 30, 49, 39, 89], 49)

console.log(getElement);

File: Basic Problems/python/linear-search.py
def linearSearch(arr, element):
  for i in range(len(arr)):
    if arr[i] == element:
      return {
        "position": i,
        "element": element
      }
  
  return "Element not found in array"

getElement = linearSearch([10, 20, 30, 49, 39, 89], 49)

print(getElement["position"])
